src/main/resources/assets/admin/common/js/ui/toolbar/Toolbar.ts

```diff
@@ -14,6 +14,7 @@ export class Toolbar
 
     protected fold: FoldButton;
     protected actions: Action[] = [];
+    protected locked: boolean;
```

Comment:
Why protected? If you need to access it from the outside, implement `isLocked` method.